apollo-router/src/plugins/telemetry/mod.rs

```diff
@@ -1245,6 +1251,58 @@ impl Telemetry {
         };
         sender.send(SingleReport::Stats(metrics));
     }
+
+    fn decode_subgraph_ftv1(context: &Context) -> (u64, HashMap<String, SingleTypeStat>) {
+        use crate::plugins::telemetry::apollo_exporter::proto::reports::trace::Node;
+
+        fn traverse(
+            per_type: &mut HashMap<String, SingleTypeStat>,
+            field_execution_weight: f64,
+            node: &Node,
+        ) {
+            let single_field_stat = per_type
+                .entry(node.parent_type.clone())
+                .or_default()
+                .per_field_stat
+                .entry(node.original_field_name.clone())
```

Comment:
I think `original_field_name` is only set if the node is aliased; the code in AS does `const fieldName = node.originalFieldName || node.responseName`.

Reply:
Yes I found the same in https://github.com/apollographql/router/pull/2541#discussion_r1096090424. Sorry I didn’t push my latest changes before your review!